src/main/java/com/revature/models/Address.java

```diff
@@ -20,17 +25,9 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	@Column(name = "address_id")
 	private int id;
-
-	@NotBlank(message = "Street cannot be blank.")
```

Comment:
Do we need to _remove_ the validation on the entity models? Wouldn't it be better to have the validation on both?

Reply:
All new submissions ought to be arriving as DTOs. I was afraid that leaving the JSR on the entities would result in people updating those with better more efficient notation and not realizing that the important ones are on the DTOs. It took us nearly 5 days to realize we had accidentally disabled JSR by adding the DTOs in, how long would it take a group not familiar with the code to realize that there is a second place that needs this information?

If we really wanted to... we could remove the noArgs constructor on the Models and only leave a constructor that builds from the DTO. That would be an almost idiot proof system and ought to only require that we rewrite a few unit tests.

Reply:
Nevermind, I doubt that would work with the JPA Repository

Reply:
Alright, that is fine.